From 15b2cf9e98b60d8972fc90c0d1f5fd5befaecfee Mon Sep 17 00:00:00 2001
From: Insei <goodmobiledevices@gmail.com>
Date: Mon, 8 Feb 2021 07:37:12 +0300
Subject: [PATCH] arch: arm: dts: rpi4: rename usb pci child node, and remove
 reg property. * in reset driver that uses this node, address for reset is
 hard-coded in
 https://github.com/torvalds/linux/blob/726eb70e0d34dc4bc4dada71f52bba8ed638431e/drivers/reset/reset-raspberrypi.c#L51

Signed-off-by: Insei <goodmobiledevices@gmail.com>
---
 arch/arm/boot/dts/bcm2711-rpi-4-b.dts | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/bcm2711-rpi-4-b.dts b/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
index 09a1182c2936..00bcd383706b 100644
--- a/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
+++ b/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
@@ -262,8 +262,7 @@ pci@1,0 {
 
 		reg = <0 0 0 0 0>;
 
-		usb@1,0 {
-			reg = <0x10000 0 0 0 0>;
+		usb {
 			resets = <&reset RASPBERRYPI_FIRMWARE_RESET_ID_USB>;
 		};
 	};
-- 
2.30.0

